components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -2435,10 +2435,10 @@ private boolean getIsB2BSSApp(List<ServiceProviderProperty> propertyList) {
         return Boolean.parseBoolean(value);
     }
 
-    private boolean getAppAccessEnabled(List<ServiceProviderProperty> propertyList) {
+    private boolean getIsAppEnabled(List<ServiceProviderProperty> propertyList) {
```

Comment:
lets be consistent is some places we have used the term "application" and in some places we use "app"